.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -22,6 +27,6 @@ jobs:
         run: |
           gh pr review --repo "${{ github.repository }}" --comment --body "auto approve" ${{ steps.find_prs.outputs.dep_pull_request }}
           gh pr review --repo "${{ github.repository }}" --approve ${{ steps.find_prs.outputs.dep_pull_request }}
-          gh pr merge --repo "${{ github.repository }}" --auto --squash ${{ steps.find_prs.outputs.dep_pull_request }}
+          gh pr merge --repo "${{ github.repository }}" --auto --squash --delete-branch ${{ steps.find_prs.outputs.dep_pull_request }}
```

Comment:
Thank you! You might be able to delete the `delete-branch` line in `evalml/.github/workflows/latest_dependency_checker.yml` in place of this (didn't seem to work anyways)